docs/ref/templates/builtins.txt

```diff
@@ -515,15 +516,6 @@ of how ``x in y`` will be interpreted::
 
 Not contained within. This is the negation of the ``in`` operator.
 
-The comparison operators cannot be 'chained' like in Python or in mathematical
```

Comment:
Another option would be to move it to the end of the [complex expressions](https://docs.djangoproject.com/en/dev/ref/templates/builtins/#complex-expressions) section.